types/hapi__mimos/index.d.ts

```diff
@@ -3,27 +3,62 @@
 // Definitions by: AJP <https://github.com/AJamesPhillips>
 //                 Silas Rech <https://github.com/lenovouser>
 //                 Linus Unnebäck <https://github.com/LinusU>
+//                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.2
```

Comment:
Which is the minimum when no version is provided? 🤔

Reply:
the tested one is 2.8 I believe, runs locally during tests and during CI
https://github.com/DefinitelyTyped/DefinitelyTyped#older-versions-of-typescript-27-and-earlier